staging/src/k8s.io/client-go/examples/README.md

```diff
@@ -0,0 +1,36 @@
+# client-go Usage Examples
+
+This directory contains examples that cover examples of various use cases
+and functionality for Go client library for Kubernetes.
+
+### Authentication
+
+- [**Authenticate in cluster**](./authenticate-in-cluster)
+- [**Authenticate out of cluster**](./authenticate-out-of-cluster)
+
+### Basics
+
+- [**Managing resources with API**](./create-update-delete-deployment): Create,
+  get, update, delete a Deployment resource.
+
+### Advanced Concepts
+
+- [**Third-party resources**](./third-party-resources): Register a custom
+  resource type with the API, create/update/query this custom type, and write a
+  controller to handle events for this custom resource type.
+- [**Work queues**](./workqueue): Create a custom controller with a Watch for Pod events,
```

Comment:
I don't understand the "Work queue" example just yet but the following 2 concepts are a bit too difficult to write here without any explanation:

- hotloop-free controller
- informer framework.

Is there a better way you think you can explain this example? 

> Create a hotloop-free controller with the rate-limited workqueue and the informer framework

I cannot infer any of this from `workqueue/README.md` which says:

```
This example demonstrates how to write a controller which follows the states
of watched resources.

It demonstrates how to:
 * combine the workqueue with a cache to a full controller
 * synchronize the controller on startup
```

:) Let me know if you have a better explanation and I will change/consolidate.